Users/signup.js

```diff
@@ -0,0 +1,44 @@
+const express = require("express")
+const router = express.Router()
+const {v4} = require("uuid")
+
+const {User} = require("./user.model.js")
+const {Cart} = require("../Carts/cart.model.js")
+const {Wishlist} = require("../Wishlists/wishlist.model.js")
+
+router.route("/")
+  .post(async (req,res) => {
+    const {name, email, password} = req.body;
+    
+    try{
+
+          const newCart = new Cart({
+            _id: v4(),
```

Comment:
Mongodb gives its own id for each object in a collection.

Reply:
I wanted to add a new cart and wishlist ids inside a user object ... as soon as a new user is signedUp .
But mongodb gives _id after the collection is created....how will I get those ids and store them inside the user object after that.
So I am using uuid to generate id(for cart & wishlist) here and at the same time stored it in user object.

Please let me know your thoughts if it make sense.